docs/index.md

```diff
@@ -1,19 +1,27 @@
-# Snitch
+# SNitch Acceleration eXtension (SNAX)
+SNitch Acceleration eXtension (SNAX) is a platform that provides the basic needs of an accelerator in a multi-core system.
 
-The Snitch project is an open-source RISC-V hardware research project of ETH Zurich and University of Bologna targeting highest possible energy-efficiency. The system is designed around a versatile and small integer core, which we call Snitch. The system is ought to be highly parameterizable and suitable for many use-cases, ranging from small, control-only cores, to large many-core system made for pure number crunching in the HPC domain.
```

Comment:
I don't see why we are removing this line, this is nice PR (as in "public relations") for our project as well.
I would add ourselves as extra contributors, not remove ETH and Bologna from the list.
This makes it a project that is less "ours" and more "everyone's" which I believe to be a good thing.

Reply:
I would just add a big and bold admonition that says you are on the KU Leuven snitch fork:

https://squidfunk.github.io/mkdocs-material/reference/admonitions/

For example:
```markdown
!!! note

    You are looking at the KULeuven fork of Snitch Cluster developed by the PULP group. 
    This fork, called SNAX, provides extra infrastructure for quickly adding tightly coupled 
    accelerators to a systems on a Snitch Cluster Level.
```
Alternatively
```markdown
!!! warning

    You are looking at the KULeuven fork of Snitch Cluster developed by the PULP group. 
    This fork, called SNAX, provides extra infrastructure for quickly adding tightly coupled 
    accelerators to a systems on a Snitch Cluster Level.
```

Reply:
I can add this instead

Reply:
I put back the initial text and added the admonition.  I still put the SNAXYou will see it in the next commits.